akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2ServerDemux.scala

```diff
@@ -69,6 +74,118 @@ class Http2ServerDemux extends GraphStage[BidiShape[Http2SubStream, FrameEvent,
     BidiShape(substreamIn, frameOut, frameIn, substreamOut)
 
   def createLogic(inheritedAttributes: Attributes): GraphStageLogic =
-    new GraphStageLogic(shape) {
+    new GraphStageLogic(shape) with BufferedOutletSupport { logic ⇒
+      case class SubStream(
```

Comment:
final case classes right away, only open up when necessary.
Boring note but should save us MiMa pains once we enable it